Pages/js/options.js

```diff
@@ -85,19 +85,19 @@ function saveWhitelist() {
   // Each domain can be from 1-63 characters and may contain alphanumeric 
   // characters and - but may not begin or end with - Each top level domain may
   // be from 2 to 9 characters and may contain alpha characters
-  var validateSubdomain = new RegExp("^(?!\-|_)[\\w\-]{1,63}","g"); //subdomains
-  var validateDomain = new RegExp("^(?!\-)[a-zA-Z0-9\-?]{1,63}$","g"); //domain
-  var validateTLD = new RegExp("^[a-zA-Z]{2,9}$","g"); //top level domain
+  var validateSubdomain = new RegExp(/^(?!\-|_)[\\w\-]{1,63}/, "g"); //subdomains
+  var validateDomain = new RegExp(/^(?!\-)[a-zA-Z0-9\-?]{1,63}$/, "g"); //domain
+  var validateTLD = new RegExp(/^[a-zA-Z]{2,9}$/, "g"); //top level domain
   
   //needed because js regex does not have look-behind
-  var notEndInHyphenOrUnder = new RegExp("[^\-_]$","g"); 
-  var notEndInHyphen = new RegExp("[^\-]$","g");
+  var notEndInHyphenOrUnder = new RegExp(/[^\-_]$/, "g"); 
+  var notEndInHyphen = new RegExp(/[^\-]$/, "g");
 
   var domain_regexp = "";  //stores regex to match validated domains
   var valid_domains = [];  //stores validated domains
   
   //iterate over entered list, split by invalid chars
-  for (var i = 0; i < domains.length; i++){ 
+  for (i = 0; i < domains.length; i++){ 
```

Comment:
Wow, thanks for the quick feedback! The reason I changed this is because Javascript doesn't use block scope (only function scope), so this variable was already "global" to the function. (This was the issue with a few other declarations as well.) So, this basically removes a redundant declaration.